package.nls.json

```diff
@@ -0,0 +1,13 @@
+{
+    "walkthrough.java.runtime": "Make sure you have at least on Java runtime installed.\n[Install New JDK](command:toSide:java.installJdk.fromWalkthrough)",
+
+    "walkthrough.java.projects": "In Java Projects explorer, you can browse your projects, view classpaths and dependencies. \n[View Java Projects](command:javaProjectExplorer.focus)",
+
+    "walkthrough.java.projects.inactive": "In Java Projects explorer, you can browse your projects, view classpaths and dependencies. To get full feautures, you should create or open a folder containing Java projects.\n[Open Command Palette](command:workbench.action.showCommands)\n__Try searching for 'Create Java Project' or 'Open Folder'__",
+
+    "walkthrough.java.debugAndTest":"The debugger will automatically find the entry point of your project, launch it and start debugging. [Learn More...](https://code.visualstudio.com/docs/java/java-debugging)\nThe [Testing View](command:testExplorer.focus) show all the test cases in your project. You can also run/debug your test cases from here. [Learn More...](https://code.visualstudio.com/docs/java/java-testing)\n[Show Run and Debug View](command:workbench.view.debug)",
+
+    "walkthrough.java.extensions": "Additional support for major toolchains, frameworks and application servers.\n[Browse Recommended Extensions](command:toSide:java.extGuide)",
```

Comment:
Install extensions to support additional frameworks such as Spring Boot, Application Servers, Keymaps and so on